### DISCUSSION TITLE
Will a physical audio interface reduce delay?

### DISCUSSION THREAD

**Discussion by YumSoup** [jamulussoftware/jamulus#1929] [Technical Support & Help] [answered]
I have a TD-11 drum kit hooked up to a laptop, which uses JACK to resample the 44.1khz to 48Khz. The TD11 only outputs at 44.1Khz. On servers with more than 15 ping my delay is in the reds and sometimes yellow.

The laptop is an old one from 2010 or so, so I was wondering if the processor speed is what causes the resample to have such delay.

If I were to buy some external equipment to resample instead of using JACK, would that reduce delay by much if at all? What sort of thing would I need to buy? Thanks for any help :)

**Comment by ann0see** [comment, marked as answer]
Hi! Thanks for opening a forum post. I must admit that I only own an USB interface and no complicated hardware ;-). 

I assume:
1. More processing power due to resampling = more CPU load —> latency
2. Hardware resamplers exist, but I don’t know of them —> Google?
3. You could get the analog audio out from your device via XLR/normal big jack connector and input it into an audio interface —> Avoids digital sample rate conversion. I did that once with an "unsupported 44.1 kHz keyboard" and it worked. 

You‘re on Linux probably?

**Reply by YumSoup** [reply]
Thanks for your reply! I'm on windows 7.
I completely forgot the possibility of analog output from the drums lol.

I suppose buying a very cheap audio interface ought to do the trick. It doesn't seem like it would have less delay than a digital resampler due to the signal having to go through more hardware, but that's just my newbie intuition.

I'll see what I can find but mostly likely I won't be able to afford much. It's good-to-knows anyway, so thanks a bunch!

**Reply by ann0see** [reply]
Don’t invest too little. Use a device with ASIO driver. ASIO4ALL adds latency - and is a pain to set up sometimes.

**Reply by ann0see** [reply]
I‘m using a UMC202HD, but maybe you’ll find cheaper devices without amps.

**Comment by QJKX** [comment]
I don't know about latency, but it should be possible to drive a virtual instrument from the midi out of the kit.
eg drumgizmo

Hopefully the kit is a class compliant usb midi device and you can just test it without buying anything.
